bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -361,9 +447,6 @@ spec:
                     memory: 50Mi
                 securityContext:
                   allowPrivilegeEscalation: false
-              securityContext:
-                runAsNonRoot: true
-                runAsUser: 2000
```

Comment:
was this added by the SDK and we've removed it? Do you still need this change even with the downgraded RBAC image version?

Reply:
@jeremyary Not sure what you meant here. What downgraded RBAC image version are you referring to?

Reply:
Can disregard, I was assuming an image version and the permissions changes were related, but that appears to not be the case.